Can't resolve 'url'

## Describe the bug

Trying to add this package to my docusaurus 2.2 project, but I get an error now when I try to start it up.

## Expected behavior

Docusaurus starts up.

## Current behavior

failure:

```
Module not found: Error: Can't resolve 'url' in '/Users/ianvs/code/defined/dn-docs/node_modules/@paloaltonetworks/postman-code-generators/codegens/csharp-restsharp/lib'
BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "url": require.resolve("url/") }'
	- install 'url'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "url": false }
client (webpack 5.75.0) compiled with 1 error
```

## Possible solution

<!--- Not obligatory, but suggest a fix/reason for the bug, -->
<!--- or ideas how to implement the addition or change -->

## Steps to reproduce

- Clone https://github.com/IanVS/repro-docusaurus-openapi-issue
- `npm i`
- `npx docusaurus gen-api-docs all`
- `npm start`

The error happens.

## Context

I'm unable to use this plugin.

## Your Environment

<!--- Include as many relevant details about the environment you experienced the bug in -->

- Version used: 1.5.2
- Environment name and version (e.g. Chrome 59, node.js 5.4, python 3.7.3): node 18, Brave browser latest.
- Operating System and version (desktop or mobile): macos
- Link to your project: https://github.com/IanVS/repro-docusaurus-openapi-issue

--- a/packages/docusaurus-theme-openapi-docs/package.json
+++ b/packages/docusaurus-theme-openapi-docs/package.json
@@ -55,6 +55,7 @@
     "file-saver": "^2.0.5",
     "immer": "^9.0.7",
     "lodash": "^4.17.20",
+    "node-polyfill-webpack-plugin": "^2.0.1",
     "process": "^0.11.10",
     "react-live": "^3.1.1",
     "react-magic-dropzone": "^1.0.1",

--- a/packages/docusaurus-theme-openapi-docs/src/index.ts
+++ b/packages/docusaurus-theme-openapi-docs/src/index.ts
@@ -8,6 +8,7 @@
 import path from "path";
 
 import type { Plugin } from "@docusaurus/types";
+
 const NodePolyfillPlugin = require("node-polyfill-webpack-plugin");
 
 export default function docusaurusThemeOpenAPI(): Plugin<void> {

--- a/packages/docusaurus-theme-openapi-docs/src/index.ts
+++ b/packages/docusaurus-theme-openapi-docs/src/index.ts
@@ -8,7 +8,7 @@
 import path from "path";
 
 import type { Plugin } from "@docusaurus/types";
-import { ProvidePlugin } from "webpack";
+const NodePolyfillPlugin = require("node-polyfill-webpack-plugin");
 
 export default function docusaurusThemeOpenAPI(): Plugin<void> {
   return {
@@ -29,18 +29,7 @@ export default function docusaurusThemeOpenAPI(): Plugin<void> {
 
     configureWebpack() {
       return {
-        plugins: [
-          new ProvidePlugin({
-            Buffer: [require.resolve("buffer/"), "Buffer"],
-            process: require.resolve("process/browser"),
-          }),
-        ],
-        resolve: {
-          fallback: {
-            buffer: require.resolve("buffer/"),
-            process: require.resolve("process/browser"),
-          },
-        },
+        plugins: [new NodePolyfillPlugin()],
       };
     },
   };

